docs/CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Test fixtures for use by clients are available for each release on the [Github r
 - ✨ Feature releases can now include multiple types of fixture tarball files from different releases that start with the same prefix ([#736](https://github.com/ethereum/execution-spec-tests/pull/736)).
 - ✨ Releases for feature eip7692 now include both Cancun and Prague based tests in the same release, in files `fixtures_eip7692.tar.gz` and `fixtures_eip7692-prague.tar.gz` respectively ([#743](https://github.com/ethereum/execution-spec-tests/pull/743)).
 - 🔀 Simplify Python project configuration and consolidate it into `pyproject.toml` ([#764](https://github.com/ethereum/execution-spec-tests/pull/764)).
+- ✨ Remove redundant tests within stable and develop fixture releases, moving them to a separate legacy release ([#788](https://github.com/ethereum/execution-spec-tests/pull/788)).
 
 ### 💥 Breaking Change
```

Comment:
Actually, could we mark this as a breaking change please?

Reply:
Nice! Yeah will do.

I'd like to understand the constantinople split that @winsvega mentioned a bit more too!